feat: add path on fuel-core run command to avoid collisions (#516)
FuelLabs/fuels-ts: docs/QUICKSTART.md

Inside `my-fuel-dapp` run;

```sh
fuel-core run --ip 127.0.0.1 --port 4000 --chain ./chainConfig.json --db-path ./.fueldb
```

> Note, that the `--db-path` param defines where the chain state should be store.

Reviewer: ```suggestion
Note, that the `--db-path` param defines where the chain state is stored.
```
